configs/setups/awiesm/awiesm-2.1.yaml

```diff
@@ -63,11 +63,12 @@ general:
         - '2.1-wiso'
         - '2.1-recom'
         choose_version:
-          '2.1':
+          '2.1-gamma':
             couplings:
-            - fesom-2.0-paleodyn+echam-6.3.05p2-concurrent_radiation-paleodyn
-            wiso_code: false
+            - fesom-2.1-paleodyn-gamma+echam-6.3.05p2-concurrent_radiation-paleodyn-wiso
+            wiso_code: true
             with_wiso: false
+            with_pico: false
```

Comment:
Why is `with_pico: false` in the `gamma` version? I thought `gamma` should have `pico` on.

Reply:
Here, no by default, but by user choice.